### PR TITLE
Feature bot 2626 rail section

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PR add rail_section

## Issue (optional)

Issue link:bot-2626

### CODE REVIEW

**Comment by pthegner** [comment]
retest this please

File: chaos/db_helper.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Copyright (c) since 2001, Kisio Digital and/or its affiliates. All rights reserved.
 
+import json
```
Comment (pthegner) [inline chaos/db_helper.py:3]: why ?

**Comment by pthegner** [comment]
Jenkins response : 
```
Only a single head is supported. The script directory has multiple heads (due to branching), which must be resolved by manually editing the revision files to form a linear sequence. Run `alembic branches` to see the divergence(s)
```

File: chaos/db_helper.py (hunk 1 above)
Comment (kun2985) [reply chaos/db_helper.py:3]: We need `json.dumps` in line 184

**Review by pthegner — APPROVED** [review]

**Comment by pthegner** [comment]
Add label do_not_merge because we have to discuss with navitia team if we want to merge now or wait a little longer.

**Comment by Fahmus** [comment]
> Add label do_not_merge because we have to discuss with navitia team if we want to merge now or wait a little longer.

We must discuss this ticket BOT-2836 before we merge

**Review by Msaglier — APPROVED** [review]

**Pull request merged**